modules/microshift-nw-router-disable.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-the-router_{context}"]
+= Disabling the router
+
+In use cases such as industrial IoT spaces where {microshift-short} pods only need to connect to southbound operational systems and northbound cloud-data systems, inbound services are not needed. Use this procedure to disable the router in such egress-only use cases.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameters in the `ingress.status` field to `Removed` as shown in the following example:
++
+.Example `config.yaml` ingress stanza
+[source,yaml]
+----
+apiServer:
+  advertiseAddress: 10.44.0.0/32
+#....
+ingress:
+  listenAddress: # <1>
+    - "10.1.235.168"
+    - "ens4"
+  ports:
+    http: 80
+    https: 443
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed
+  status: Removed # <2>
+# ....
+----
+<1> You must either remove or configure the `listenAddress` stanza when using the `config.yaml` file. Do not leave the field empty or the cluster crashes on restart.
```

Comment:
This may fit better in the `microshift-nw-config-IPs.adoc`. Swap `cluster` with `{microshift-short}`